### ISSUE TITLE
In case of older libvirt version driver version check doesn't provide proper detail.

### ISSUE THREAD

**Issue by praveenkumar** [code-ready/crc#475]
If a user on older linux system and have libvirt < 3.4.0 then installation of driver succeed but the actual check failed without much details to the user. If a user manually try to check the version then it have more info.

```
# crc start -b .crc/crc_libvirt_4.1.9.crcbundle 
INFO Checking if oc binary is cached              
INFO Checking if Virtualization is enabled        
INFO Checking if KVM is enabled                   
INFO Checking if libvirt is installed             
INFO Checking if user is part of libvirt group    
INFO Checking if libvirt is enabled               
INFO Checking if libvirt daemon is running        
INFO Checking if crc-driver-libvirt is installed  
FATA exit status 1 

# crc-driver-libvirt -v
crc-driver-libvirt: /lib64/libvirt.so.0: version `LIBVIRT_3.4.0' not found (required by crc-driver-libvirt)
```

**Comment by cfergeau** [comment]
On which system did it happen? RHEL 7.5 and newer have a libvirt more recent than 3.4.0. In addition to printing the error, we could catch this at crc setup time with a libvirt version check (though detecting which minimum version a given crc-driver-libvirt binary needs is a bit tricky/ugly).

**Comment by gbraad** [comment]
This would in general not be an issue for a new user.
Does this need to included in a new beta version?

**Comment by praveenkumar** [comment]
@gbraad no, until a new user is really using an old version of libvirt.

fixed by #476

**Comment by gbraad** [comment]
I saw the merge... so we can postpone this and aim for after the beta, towards GA.
